digitaltwins/samples/device/javascript/sample_device.js

```diff
@@ -23,19 +31,19 @@ const propertyUpdateHandler = async (component, propertyName, reportedValue, des
 
 const commandHandler = (request, response) => {
```

Comment:
@YoDaMa this statement throws a runtime exception, looks you need to mark the handler as async.

```
/workspace/azure-iot-sdk-node/digitaltwins/samples/device/javascript/sample_device.js:26
  await response.acknowledge(200, 'helpful response text');
  ^^^^^

SyntaxError: await is only valid in async function
```